src/main/java/com/iota/iri/service/validation/TransactionValidator.java

```diff
@@ -449,8 +379,10 @@ private boolean checkApproovee(TransactionViewModel approovee) throws Exception
         return approovee.isSolid();
     }
 
-    @VisibleForTesting
-    boolean isNewSolidTxSetsEmpty () {
+    /**
+     * Exclusively used in the {@link TansactionValidatorTest}
+     */
+    public boolean isNewSolidTxSetsEmpty () {
```

Comment:
If so, why did you change the signature?

Reply:
For whatever reason it was firing off a warning on my ide when it was this way, just changed it back without incident though.